app/helpers/hyrax_helper.rb

```diff
@@ -15,6 +15,7 @@ def link_to_html(field, show_link = true, truncate = false)
       text = field
     end
 
+    text = Loofah.fragment(text).scrub!(:whitewash).to_s
```

Comment:
NOTE: I've chosen the strictest setting -- `:whitewash`. This will take out the `href` in `<a>` tags.  We can also try `:prune` if you want it less strict, but I thought to err on the side of caution.

Reply:
Or, maybe we can make this configurable.